todolist/src/common/GNB.jsx

```diff
@@ -0,0 +1,33 @@
+import { AppBar, Box, Button, Toolbar, Typography } from '@mui/material';
+import React from 'react';
+import { LinkToRegist, LinkToMain } from 'component/location/LinkTo';
+import { useNavigate } from 'react-router-dom';
+import HomeIcon from '@mui/icons-material/Home';
+
+const GNB = () => {
+  const navigator = useNavigate();
+
+  const linkToMainFn = () => {
+    navigator('/');
+  };
+  return (
+    <Box>
+      <AppBar position='static'>
+        <Toolbar>
+          <Typography
+            variant='h6'
+            component='div'
+            sx={{ flexGrow: 1 }}
+            onClick={linkToMainFn}
+          >
+            <Button color='inherit'>Journey</Button>
+          </Typography>
```

Comment:
```suggestion
          <Typography
            variant='h6'
            component='h6'
            sx={{ flexGrow: 1 }}
            onClick={linkToMainFn}
          >
            <Button color='inherit'>Journey</Button>
          </Typography>
```
시멘틱하게 태그 설정 하는 게 더 좋아 보입니다.